README.md

```diff
@@ -1,15 +1,38 @@
-## Welcome friend!
```

Comment:
The existing README.md should not be overridden.
You can add things though.

Reply:
On it